tendrl/ceph_integration/manager/__init__.py

```diff
@@ -84,6 +86,30 @@ def main():
     m = CephIntegrationManager()
     m.start()
 
+    # Save the monitor secret
+    gevent.sleep(5)
+    out, err, rc = cmd_utils.Command(
```

Comment:
The monitor key is something only once we need to save after import is successful. We dont need to keep this syncing again and again as this doesnt change once cluster is created.

If you are particular about sds_sync, may be we can add the code there with a check. If monitor doesnt exists already, we would sync else not.

Comments?